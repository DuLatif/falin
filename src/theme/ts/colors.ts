export const neutral = {
  25: "#f9fafb",
  50: "#f8fafc",
  100: "#f1f5f9",
  200: "#e2e8f0",
  300: "#cbd5e1",
  400: "#94a3b8",
  500: "#64748b",
  600: "#475569",
  700: "#334155",
  800: "#1e293b",
  900: "#0f172a",
}

export const primary = {
  25: "#f5f8ff",
  50: "#eff6ff",
  100: "#dbeafe",
  200: "#bfdbfe",
  300: "#93c5fd",
  400: "#60a5fa",
  500: "#3b82f6",
  600: "#2563eb",
  700: "#1d4ed8",
  800: "#1e40af",
  900: "#1e3a8a",
}

export const success = {
  25: "#f6fef9",
  50: "#ecfdf3",
  100: "#d1fadf",
  200: "#a6f4c5",
  300: "#6ce9a6",
  400: "#32d583",
  500: "#12b76a",
  600: "#039855",
  700: "#027a48",
  800: "#05603a",
  900: "#054f31",
}

export const info = {
  25: "#fafeff",
  50: "#f0f9ff",
  100: "#e0f2fe",
  200: "#bae6fd",
  300: "#7dd3fc",
  400: "#38bdf8",
  500: "#0ea5e9",
  600: "#0284c7",
  700: "#0369a1",
  800: "#075985",
  900: "#0c4a6e",
}

export const warning = {
  25: "#fffcf5",
  50: "#fff7ed",
  100: "#ffedd5",
  200: "#fed7aa",
  300: "#fdba74",
  400: "#fb923c",
  500: "#f97316",
  600: "#ea580c",
  700: "#c2410c",
  800: "#9a3412",
  900: "#7c2d12",
}

export const danger = {
  25: "#fffbfa",
  50: "#fef2f2",
  100: "#fee2e2",
  200: "#fecaca",
  300: "#fca5a5",
  400: "#f87171",
  500: "#ef4444",
  600: "#dc2626",
  700: "#b91c1c",
  800: "#991b1b",
  900: "#7f1d1d",
}
