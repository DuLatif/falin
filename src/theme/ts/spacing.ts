export const spacing = [0, 4, 8, 12, 16, 20, 24, 32, 40, 48, 64, 80, 96, 128, 160, 192, 224, 256]
