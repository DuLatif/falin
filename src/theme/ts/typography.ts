import { TypographyOptions } from "@mui/material/styles/createTypography"

declare module "@mui/material/styles" {
  interface TypographyOptions {
    fontWeightSemiBold: React.CSSProperties["fontWeight"]
  }

  interface TypographyVariantsOptions {
    fontWeightSemiBold?: React.CSSProperties["fontWeight"]
  }
}

declare module "@mui/material/Typography" {
  interface TypographyPropsFontWeightOverrides {
    semiBold: true
  }
}

export const typography: TypographyOptions = {
  fontFamily: "Inter",
  fontSize: 16,
  fontWeightRegular: 400,
  fontWeightMedium: 500,
  fontWeightSemiBold: 600,
  fontWeightBold: 700,
  h1: {
    fontSize: "4.5rem",
    lineHeight: "5.625rem",
    letterSpacing: "-2%",
  },
  h2: {
    fontSize: "3.75rem",
    lineHeight: "4.5rem",
    letterSpacing: "-2%",
  },
  h3: {
    fontSize: "3rem",
    lineHeight: "3.75rem",
    letterSpacing: "-2%",
  },
  h4: {
    fontSize: "2.25rem",
    lineHeight: "2.75rem",
    letterSpacing: "-2%",
  },
  h5: {
    fontSize: "1.875rem",
    lineHeight: "2.375rem",
  },
  h6: {
    fontSize: "1.5rem",
    lineHeight: "2rem",
  },
  subtitle1: {
    fontSize: "1.25rem",
    lineHeight: "1.875rem",
  },
  subtitle2: {
    fontSize: "1.125rem",
    lineHeight: "1.75rem",
  },
  body1: {
    fontSize: "1rem",
    lineHeight: "1.5rem",
  },
  body2: {
    fontSize: "0.875rem",
    lineHeight: "1.25rem",
  },
  caption: {
    fontSize: "0.75rem",
    lineHeight: "1.125rem",
  },
  overline: {
    fontSize: "0.625rem",
    lineHeight: "1rem",
  },
}
