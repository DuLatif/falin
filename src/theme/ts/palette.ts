import { PaletteOptions } from "@mui/material/styles"
import { neutral, primary, danger, warning, success, info } from "./colors"

export const palette: PaletteOptions = {
  primary: {
    light: primary[400],
    main: primary[500],
    dark: primary[600],
  },
  error: {
    light: danger[400],
    main: danger[500],
    dark: danger[600],
  },
  warning: {
    light: warning[400],
    main: warning[500],
    dark: warning[600],
  },
  success: {
    light: success[400],
    main: success[500],
    dark: success[600],
  },
  info: {
    light: info[400],
    main: info[500],
    dark: info[600],
  },
  grey: {
    100: neutral[100],
    200: neutral[200],
    300: neutral[300],
    400: neutral[400],
    500: neutral[500],
    600: neutral[600],
    700: neutral[700],
    800: neutral[800],
    900: neutral[900],
  },
  text: {
    primary: neutral[700],
    secondary: neutral[500],
  },
}
