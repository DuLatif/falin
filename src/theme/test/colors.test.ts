import { describe, expect, test } from "vitest"
import { danger, info, neutral, primary, success, warning } from "../ts/colors"
import { readFileSync } from "fs"

describe("Theme Color", () => {
  test("should have the complete level for each color", () => {
    const totalLevel = 11
    const getLength = (color: object) => Object.keys(color).length
    expect(getLength(neutral)).toBe(totalLevel)
    expect(getLength(primary)).toBe(totalLevel)
    expect(getLength(success)).toBe(totalLevel)
    expect(getLength(info)).toBe(totalLevel)
    expect(getLength(warning)).toBe(totalLevel)
    expect(getLength(danger)).toBe(totalLevel)
  })

  test("should match neutral snapshot", () => {
    expect(neutral).toMatchSnapshot()
  })

  test("should match primary snapshot", () => {
    expect(primary).toMatchSnapshot()
  })

  test("should match success snapshot", () => {
    expect(success).toMatchSnapshot()
  })

  test("should match info snapshot", () => {
    expect(info).toMatchSnapshot()
  })

  test("should match warning snapshot", () => {
    expect(warning).toMatchSnapshot()
  })

  test("should match danger snapshot", () => {
    expect(danger).toMatchSnapshot()
  })

  test("should match scss variables snapshot", () => {
    expect(readFileSync(__dirname + "./../scss/colors.scss")).toMatchSnapshot()
  })
})
