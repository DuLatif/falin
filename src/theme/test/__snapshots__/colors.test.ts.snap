// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Theme Color > should match danger snapshot 1`] = `
{
  "100": "#fee2e2",
  "200": "#fecaca",
  "25": "#fffbfa",
  "300": "#fca5a5",
  "400": "#f87171",
  "50": "#fef2f2",
  "500": "#ef4444",
  "600": "#dc2626",
  "700": "#b91c1c",
  "800": "#991b1b",
  "900": "#7f1d1d",
}
`;

exports[`Theme Color > should match info snapshot 1`] = `
{
  "100": "#e0f2fe",
  "200": "#bae6fd",
  "25": "#fafeff",
  "300": "#7dd3fc",
  "400": "#38bdf8",
  "50": "#f0f9ff",
  "500": "#0ea5e9",
  "600": "#0284c7",
  "700": "#0369a1",
  "800": "#075985",
  "900": "#0c4a6e",
}
`;

exports[`Theme Color > should match neutral snapshot 1`] = `
{
  "100": "#f1f5f9",
  "200": "#e2e8f0",
  "25": "#f9fafb",
  "300": "#cbd5e1",
  "400": "#94a3b8",
  "50": "#f8fafc",
  "500": "#64748b",
  "600": "#475569",
  "700": "#334155",
  "800": "#1e293b",
  "900": "#0f172a",
}
`;

exports[`Theme Color > should match primary snapshot 1`] = `
{
  "100": "#dbeafe",
  "200": "#bfdbfe",
  "25": "#f5f8ff",
  "300": "#93c5fd",
  "400": "#60a5fa",
  "50": "#eff6ff",
  "500": "#3b82f6",
  "600": "#2563eb",
  "700": "#1d4ed8",
  "800": "#1e40af",
  "900": "#1e3a8a",
}
`;

exports[`Theme Color > should match scss variables snapshot 1`] = `
{
  "data": [
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    50,
    53,
    58,
    32,
    35,
    102,
    50,
    102,
    52,
    102,
    55,
    59,
    13,
    10,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    53,
    48,
    58,
    32,
    35,
    102,
    56,
    102,
    97,
    102,
    99,
    59,
    13,
    10,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    49,
    48,
    48,
    58,
    32,
    35,
    102,
    49,
    102,
    53,
    102,
    57,
    59,
    13,
    10,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    50,
    48,
    48,
    58,
    32,
    35,
    101,
    50,
    101,
    56,
    102,
    48,
    59,
    13,
    10,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    51,
    48,
    48,
    58,
    32,
    35,
    99,
    98,
    100,
    53,
    101,
    49,
    59,
    13,
    10,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    52,
    48,
    48,
    58,
    32,
    35,
    57,
    52,
    97,
    51,
    98,
    56,
    59,
    13,
    10,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    53,
    48,
    48,
    58,
    32,
    35,
    54,
    52,
    55,
    52,
    56,
    98,
    59,
    13,
    10,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    54,
    48,
    48,
    58,
    32,
    35,
    52,
    55,
    53,
    53,
    54,
    57,
    59,
    13,
    10,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    55,
    48,
    48,
    58,
    32,
    35,
    51,
    51,
    52,
    49,
    53,
    53,
    59,
    13,
    10,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    56,
    48,
    48,
    58,
    32,
    35,
    49,
    101,
    50,
    57,
    51,
    98,
    59,
    13,
    10,
    36,
    110,
    101,
    117,
    116,
    114,
    97,
    108,
    45,
    57,
    48,
    48,
    58,
    32,
    35,
    48,
    102,
    49,
    55,
    50,
    97,
    59,
    13,
    10,
    13,
    10,
    36,
    112,
    114,
    105,
    109,
    97,
    114,
    121,
    45,
    50,
    53,
    58,
    32,
    35,
    102,
    53,
    102,
    56,
    102,
    102,
    59,
    13,
    10,
    36,
    112,
    114,
    105,
    109,
    97,
    114,
    121,
    45,
    53,
    48,
    58,
    32,
    35,
    101,
    102,
    102,
    54,
    102,
    102,
    59,
    13,
    10,
    36,
    112,
    114,
    105,
    109,
    97,
    114,
    121,
    45,
    49,
    48,
    48,
    58,
    32,
    35,
    100,
    98,
    101,
    97,
    102,
    101,
    59,
    13,
    10,
    36,
    112,
    114,
    105,
    109,
    97,
    114,
    121,
    45,
    50,
    48,
    48,
    58,
    32,
    35,
    98,
    102,
    100,
    98,
    102,
    101,
    59,
    13,
    10,
    36,
    112,
    114,
    105,
    109,
    97,
    114,
    121,
    45,
    51,
    48,
    48,
    58,
    32,
    35,
    57,
    51,
    99,
    53,
    102,
    100,
    59,
    13,
    10,
    36,
    112,
    114,
    105,
    109,
    97,
    114,
    121,
    45,
    52,
    48,
    48,
    58,
    32,
    35,
    54,
    48,
    97,
    53,
    102,
    97,
    59,
    13,
    10,
    36,
    112,
    114,
    105,
    109,
    97,
    114,
    121,
    45,
    53,
    48,
    48,
    58,
    32,
    35,
    51,
    98,
    56,
    50,
    102,
    54,
    59,
    13,
    10,
    36,
    112,
    114,
    105,
    109,
    97,
    114,
    121,
    45,
    54,
    48,
    48,
    58,
    32,
    35,
    50,
    53,
    54,
    51,
    101,
    98,
    59,
    13,
    10,
    36,
    112,
    114,
    105,
    109,
    97,
    114,
    121,
    45,
    55,
    48,
    48,
    58,
    32,
    35,
    49,
    100,
    52,
    101,
    100,
    56,
    59,
    13,
    10,
    36,
    112,
    114,
    105,
    109,
    97,
    114,
    121,
    45,
    56,
    48,
    48,
    58,
    32,
    35,
    49,
    101,
    52,
    48,
    97,
    102,
    59,
    13,
    10,
    36,
    112,
    114,
    105,
    109,
    97,
    114,
    121,
    45,
    57,
    48,
    48,
    58,
    32,
    35,
    49,
    101,
    51,
    97,
    56,
    97,
    59,
    13,
    10,
    13,
    10,
    36,
    115,
    117,
    99,
    99,
    101,
    115,
    115,
    45,
    50,
    53,
    58,
    32,
    35,
    102,
    54,
    102,
    101,
    102,
    57,
    59,
    13,
    10,
    36,
    115,
    117,
    99,
    99,
    101,
    115,
    115,
    45,
    53,
    48,
    58,
    32,
    35,
    101,
    99,
    102,
    100,
    102,
    51,
    59,
    13,
    10,
    36,
    115,
    117,
    99,
    99,
    101,
    115,
    115,
    45,
    49,
    48,
    48,
    58,
    32,
    35,
    100,
    49,
    102,
    97,
    100,
    102,
    59,
    13,
    10,
    36,
    115,
    117,
    99,
    99,
    101,
    115,
    115,
    45,
    50,
    48,
    48,
    58,
    32,
    35,
    97,
    54,
    102,
    52,
    99,
    53,
    59,
    13,
    10,
    36,
    115,
    117,
    99,
    99,
    101,
    115,
    115,
    45,
    51,
    48,
    48,
    58,
    32,
    35,
    54,
    99,
    101,
    57,
    97,
    54,
    59,
    13,
    10,
    36,
    115,
    117,
    99,
    99,
    101,
    115,
    115,
    45,
    52,
    48,
    48,
    58,
    32,
    35,
    51,
    50,
    100,
    53,
    56,
    51,
    59,
    13,
    10,
    36,
    115,
    117,
    99,
    99,
    101,
    115,
    115,
    45,
    53,
    48,
    48,
    58,
    32,
    35,
    49,
    50,
    98,
    55,
    54,
    97,
    59,
    13,
    10,
    36,
    115,
    117,
    99,
    99,
    101,
    115,
    115,
    45,
    54,
    48,
    48,
    58,
    32,
    35,
    48,
    51,
    57,
    56,
    53,
    53,
    59,
    13,
    10,
    36,
    115,
    117,
    99,
    99,
    101,
    115,
    115,
    45,
    55,
    48,
    48,
    58,
    32,
    35,
    48,
    50,
    55,
    97,
    52,
    56,
    59,
    13,
    10,
    36,
    115,
    117,
    99,
    99,
    101,
    115,
    115,
    45,
    56,
    48,
    48,
    58,
    32,
    35,
    48,
    53,
    54,
    48,
    51,
    97,
    59,
    13,
    10,
    36,
    115,
    117,
    99,
    99,
    101,
    115,
    115,
    45,
    57,
    48,
    48,
    58,
    32,
    35,
    48,
    53,
    52,
    102,
    51,
    49,
    59,
    13,
    10,
    13,
    10,
    36,
    105,
    110,
    102,
    111,
    45,
    50,
    53,
    58,
    32,
    35,
    102,
    97,
    102,
    101,
    102,
    102,
    59,
    13,
    10,
    36,
    105,
    110,
    102,
    111,
    45,
    53,
    48,
    58,
    32,
    35,
    102,
    48,
    102,
    57,
    102,
    102,
    59,
    13,
    10,
    36,
    105,
    110,
    102,
    111,
    45,
    49,
    48,
    48,
    58,
    32,
    35,
    101,
    48,
    102,
    50,
    102,
    101,
    59,
    13,
    10,
    36,
    105,
    110,
    102,
    111,
    45,
    50,
    48,
    48,
    58,
    32,
    35,
    98,
    97,
    101,
    54,
    102,
    100,
    59,
    13,
    10,
    36,
    105,
    110,
    102,
    111,
    45,
    51,
    48,
    48,
    58,
    32,
    35,
    55,
    100,
    100,
    51,
    102,
    99,
    59,
    13,
    10,
    36,
    105,
    110,
    102,
    111,
    45,
    52,
    48,
    48,
    58,
    32,
    35,
    51,
    56,
    98,
    100,
    102,
    56,
    59,
    13,
    10,
    36,
    105,
    110,
    102,
    111,
    45,
    53,
    48,
    48,
    58,
    32,
    35,
    48,
    101,
    97,
    53,
    101,
    57,
    59,
    13,
    10,
    36,
    105,
    110,
    102,
    111,
    45,
    54,
    48,
    48,
    58,
    32,
    35,
    48,
    50,
    56,
    52,
    99,
    55,
    59,
    13,
    10,
    36,
    105,
    110,
    102,
    111,
    45,
    55,
    48,
    48,
    58,
    32,
    35,
    48,
    51,
    54,
    57,
    97,
    49,
    59,
    13,
    10,
    36,
    105,
    110,
    102,
    111,
    45,
    56,
    48,
    48,
    58,
    32,
    35,
    48,
    55,
    53,
    57,
    56,
    53,
    59,
    13,
    10,
    36,
    105,
    110,
    102,
    111,
    45,
    57,
    48,
    48,
    58,
    32,
    35,
    48,
    99,
    52,
    97,
    54,
    101,
    59,
    13,
    10,
    13,
    10,
    36,
    119,
    97,
    114,
    110,
    105,
    110,
    103,
    45,
    50,
    53,
    58,
    32,
    35,
    102,
    102,
    102,
    99,
    102,
    53,
    59,
    13,
    10,
    36,
    119,
    97,
    114,
    110,
    105,
    110,
    103,
    45,
    53,
    48,
    58,
    32,
    35,
    102,
    102,
    102,
    55,
    101,
    100,
    59,
    13,
    10,
    36,
    119,
    97,
    114,
    110,
    105,
    110,
    103,
    45,
    49,
    48,
    48,
    58,
    32,
    35,
    102,
    102,
    101,
    100,
    100,
    53,
    59,
    13,
    10,
    36,
    119,
    97,
    114,
    110,
    105,
    110,
    103,
    45,
    50,
    48,
    48,
    58,
    32,
    35,
    102,
    101,
    100,
    55,
    97,
    97,
    59,
    13,
    10,
    36,
    119,
    97,
    114,
    110,
    105,
    110,
    103,
    45,
    51,
    48,
    48,
    58,
    32,
    35,
    102,
    100,
    98,
    97,
    55,
    52,
    59,
    13,
    10,
    36,
    119,
    97,
    114,
    110,
    105,
    110,
    103,
    45,
    52,
    48,
    48,
    58,
    32,
    35,
    102,
    98,
    57,
    50,
    51,
    99,
    59,
    13,
    10,
    36,
    119,
    97,
    114,
    110,
    105,
    110,
    103,
    45,
    53,
    48,
    48,
    58,
    32,
    35,
    102,
    57,
    55,
    51,
    49,
    54,
    59,
    13,
    10,
    36,
    119,
    97,
    114,
    110,
    105,
    110,
    103,
    45,
    54,
    48,
    48,
    58,
    32,
    35,
    101,
    97,
    53,
    56,
    48,
    99,
    59,
    13,
    10,
    36,
    119,
    97,
    114,
    110,
    105,
    110,
    103,
    45,
    55,
    48,
    48,
    58,
    32,
    35,
    99,
    50,
    52,
    49,
    48,
    99,
    59,
    13,
    10,
    36,
    119,
    97,
    114,
    110,
    105,
    110,
    103,
    45,
    56,
    48,
    48,
    58,
    32,
    35,
    57,
    97,
    51,
    52,
    49,
    50,
    59,
    13,
    10,
    36,
    119,
    97,
    114,
    110,
    105,
    110,
    103,
    45,
    57,
    48,
    48,
    58,
    32,
    35,
    55,
    99,
    50,
    100,
    49,
    50,
    59,
    13,
    10,
    13,
    10,
    36,
    100,
    97,
    110,
    103,
    101,
    114,
    45,
    50,
    53,
    58,
    32,
    35,
    102,
    102,
    102,
    98,
    102,
    97,
    59,
    13,
    10,
    36,
    100,
    97,
    110,
    103,
    101,
    114,
    45,
    53,
    48,
    58,
    32,
    35,
    102,
    101,
    102,
    50,
    102,
    50,
    59,
    13,
    10,
    36,
    100,
    97,
    110,
    103,
    101,
    114,
    45,
    49,
    48,
    48,
    58,
    32,
    35,
    102,
    101,
    101,
    50,
    101,
    50,
    59,
    13,
    10,
    36,
    100,
    97,
    110,
    103,
    101,
    114,
    45,
    50,
    48,
    48,
    58,
    32,
    35,
    102,
    101,
    99,
    97,
    99,
    97,
    59,
    13,
    10,
    36,
    100,
    97,
    110,
    103,
    101,
    114,
    45,
    51,
    48,
    48,
    58,
    32,
    35,
    102,
    99,
    97,
    53,
    97,
    53,
    59,
    13,
    10,
    36,
    100,
    97,
    110,
    103,
    101,
    114,
    45,
    52,
    48,
    48,
    58,
    32,
    35,
    102,
    56,
    55,
    49,
    55,
    49,
    59,
    13,
    10,
    36,
    100,
    97,
    110,
    103,
    101,
    114,
    45,
    53,
    48,
    48,
    58,
    32,
    35,
    101,
    102,
    52,
    52,
    52,
    52,
    59,
    13,
    10,
    36,
    100,
    97,
    110,
    103,
    101,
    114,
    45,
    54,
    48,
    48,
    58,
    32,
    35,
    100,
    99,
    50,
    54,
    50,
    54,
    59,
    13,
    10,
    36,
    100,
    97,
    110,
    103,
    101,
    114,
    45,
    55,
    48,
    48,
    58,
    32,
    35,
    98,
    57,
    49,
    99,
    49,
    99,
    59,
    13,
    10,
    36,
    100,
    97,
    110,
    103,
    101,
    114,
    45,
    56,
    48,
    48,
    58,
    32,
    35,
    57,
    57,
    49,
    98,
    49,
    98,
    59,
    13,
    10,
    36,
    100,
    97,
    110,
    103,
    101,
    114,
    45,
    57,
    48,
    48,
    58,
    32,
    35,
    55,
    102,
    49,
    100,
    49,
    100,
    59,
    13,
    10,
  ],
  "type": "Buffer",
}
`;

exports[`Theme Color > should match success snapshot 1`] = `
{
  "100": "#d1fadf",
  "200": "#a6f4c5",
  "25": "#f6fef9",
  "300": "#6ce9a6",
  "400": "#32d583",
  "50": "#ecfdf3",
  "500": "#12b76a",
  "600": "#039855",
  "700": "#027a48",
  "800": "#05603a",
  "900": "#054f31",
}
`;

exports[`Theme Color > should match warning snapshot 1`] = `
{
  "100": "#ffedd5",
  "200": "#fed7aa",
  "25": "#fffcf5",
  "300": "#fdba74",
  "400": "#fb923c",
  "50": "#fff7ed",
  "500": "#f97316",
  "600": "#ea580c",
  "700": "#c2410c",
  "800": "#9a3412",
  "900": "#7c2d12",
}
`;
